src/SME.SGP.Aplicacao/Commands/EncaminhamentoAEE/EnviarParaAnaliseEncaminhamentoAEE/EnviarParaAnaliseEncaminhamentoAEECommandHandler.cs

```diff
@@ -1,31 +1,55 @@
 ﻿using MediatR;
+using SME.SGP.Aplicacao.Integracoes;
 using SME.SGP.Dominio;
 using SME.SGP.Dominio.Interfaces;
+using SME.SGP.Infra;
 using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace SME.SGP.Aplicacao
 {
     public class EnviarParaAnaliseEncaminhamentoAEECommandHandler : IRequestHandler<EnviarParaAnaliseEncaminhamentoAEECommand, bool>
     {
-        public IMediator mediator { get; }
-        public IRepositorioEncaminhamentoAEE repositorioEncaminhamentoAEE { get; }
+        private readonly IMediator mediator;
+        private readonly IRepositorioEncaminhamentoAEE repositorioEncaminhamentoAEE;
+        private readonly IServicoEol servicoEol;
 
-        public EnviarParaAnaliseEncaminhamentoAEECommandHandler(IMediator mediator, IRepositorioEncaminhamentoAEE repositorioEncaminhamentoAEE)
+
+
+        public EnviarParaAnaliseEncaminhamentoAEECommandHandler(IMediator mediator, IRepositorioEncaminhamentoAEE repositorioEncaminhamentoAEE, IServicoEol servicoEol)
         {
             this.mediator = mediator ?? throw new ArgumentNullException(nameof(mediator));
             this.repositorioEncaminhamentoAEE = repositorioEncaminhamentoAEE ?? throw new ArgumentNullException(nameof(repositorioEncaminhamentoAEE));
+            this.servicoEol = servicoEol ?? throw new ArgumentNullException(nameof(servicoEol));
         }
 
         public async Task<bool> Handle(EnviarParaAnaliseEncaminhamentoAEECommand request, CancellationToken cancellationToken)
         {
             var encaminhamentoAEE = await mediator.Send(new ObterEncaminhamentoAEEComTurmaPorIdQuery(request.EncaminhamentoId));
 
+            var turma = await mediator.Send(new ObterTurmaComUeEDrePorIdQuery(encaminhamentoAEE.TurmaId));
+
+            if (turma == null)
+                throw new NegocioException("turma não encontrada");           
+
             if (encaminhamentoAEE == null)
                 throw new NegocioException("O encaminhamento informado não foi encontrado");
 
-            encaminhamentoAEE.Situacao = Dominio.Enumerados.SituacaoAEE.Analise;
+            encaminhamentoAEE.Situacao = Dominio.Enumerados.SituacaoAEE.AtribuicaoResponsavel;
+
+            IEnumerable<Guid> perfis = new List<Guid>() { Perfis.PERFIL_PAEE };
+
+            var funciorarioPAEE = await servicoEol.ObterFuncionariosDreUePorPerfis(turma.UeId.ToString(), perfis);
```

Comment:
turma.UeId é o id da ue no SGP. Tem que passar o código do EOL pra pesquisa
```suggestion
            var funciorarioPAEE = await servicoEol.ObterFuncionariosDreUePorPerfis(turma.Ue.CodigoUe, perfis);
```